src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export interface Figure {
+  getArea: () => number;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape: string;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
[CODE KNOWLEDGE] - In TypeScript, declaring constructor parameters with an access modifier implicitly creates and assigns properties. You don't need to assign them manually inside the constructor. Remove the manual assignments to `this.shape`.